src/main/java/edu/wpi/grip/ui/util/CSSElementUtility.java

```diff
@@ -0,0 +1,59 @@
+package edu.wpi.grip.ui.util;
+
+import edu.wpi.grip.core.Connection;
+import edu.wpi.grip.core.Operation;
+import edu.wpi.grip.core.Step;
+
+public final class CSSElementUtility {
```

Comment:
I like that this isn't in a random default method in the core package anymore, but there has to be a better name for this class.  Something that makes it clear that this is for looking up elements in unit tests.  Also documentation that makes that clear.